Resources/views/LineTimetable/schedule.html.twig

```diff
@@ -0,0 +1,71 @@
+{% extends "CanalTPMttBundle::container_and_menu.html.twig" %}
+
+{% import "CanalTPMttBundle:Layouts:macros.html.twig" as macros %}
+{% import "CanalTPMttBundle::macros.html.twig" as rendering %}
+
+{% set externalRouteId = schedule|keys|first %}
+{% set columnsLimit = 15 %}
+
+{% block breadcrumb %}
+    <li>{{ 'menu.edit_timetables'|trans }}</li>
+{% endblock %}
+
+{% block main_content %}
+    <h1>{{ 'line_timetable.title.schedule'|trans({}, 'default') }} {{ rendering.lineNumber(externalLineData, false) }}</h1>
+    <br>
+    <div class="row top-menu">
+        <div class="col-md-2">
+            <a href="{{ app.request.headers.get('referer') }}" class="btn btn-default">
+                <span class="glyphicon glyphicon-chevron-left"></span> {{ 'global.go_back'|trans }}
+            </a>
+        </div>
+        <div class="col-md-6">
+            <select id="select-route" class="form-control">
+                {% for routeId, route in schedule %}
+                    <option value="{{ loop.index }}" {% if routeId == externalRouteId %}"selected='selected'"{% endif %}>{{ route.direction }}</option>
+                {% endfor %}
+            </select>
+        </div>
+    </div>
+
+    {% for routeId, route in schedule %}
+        <ul class="nav nav-tabs calendars-tab" id="{{ loop.index }}">
+            {% for calendar in route.calendars %}
+                <li{% if loop.first %} class="active"{% endif %}>
+                    <a href="#{{ loop.parent.loop.index }}-calendar-{{ calendar.id }}" data-toggle="tab">{{ calendar.name }}</a>
+                </li>
+            {% endfor %}
+        </ul>
+
+        <div class="tab-content calendars-tab" id="{{ loop.index }}">
```

Comment:
I mean you have the same id here https://github.com/CanalTP/MttBundle/pull/85#discussion-diff-45750488R32

an id is unique